shuriken/mjolnir/include/mjolnir/MjolnIRDialect.td

```diff
@@ -75,4 +75,19 @@ def MjolnIR_InvokeTypeAttr : I32EnumAttr<
   let cppNamespace = "::mlir::shuriken::MjolnIR";
 }
 
+//===----------------------------------------------------------------------===//
+// MjolnIR_FieldTypeAttr
+// Attribute used for specifying the type of load/get of a field (instance or static)
+//===----------------------------------------------------------------------===//
+def MjolnIR_FieldTypeAttr : I32EnumAttr<
+    "FieldAccessType", "Different types of load/get of a field",
+    [
+        I32EnumAttrCase<"INSTANCE", 0, "instance">,
+        I32EnumAttrCase<"STATIC", 1, "static">,
+        I32EnumAttrCase<"NONE", 99, "none">
+    ]> {
+    let cppNamespace = "::mlir::shuriken::MjolnIR";
```

Comment:
I think we have a field type already from 45 to 58?

Reply:
should we also have a case NONE? doxygen docs above in line 80 says either Instance or Static

Reply:
This is not a type, this is an attribute, probably the field type will be removed since it is not used. Attributes are enums in C++.